### PR TITLE
Stack2

### PR DESCRIPTION
**Pull request by pmuir**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds stack tab to Space->Analyze

* **What is the new behavior (if this is a feature change)?**
Adds stack tab. Note that this should in the long run be an extension to fabric8-ui, or maybe moved to the runtime console.

* **Other information**:

### CODE REVIEW

File: src/app/analyze/stack/stack-analyses.model.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export class StackAnalysesModel {
+  ecosystem: string;
+  package: string;
```
Comment (joshuawilson) [inline src/app/analyze/stack/stack-analyses.model.ts]: "package" is used as a name and is a reserved word

File: src/app/analyze/stack/stack-details/stack-details.component.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+  </div>
+
+
+  <div id="result-body" class="container widget-row widget-span" style="display:block">
+<!-- <div class="widget-span"> -->
+<!-- <br> -->
+<!-- <div class="stack_info_container"> -->
+
+<div class="row">
```
Comment (joshuawilson) [inline src/app/analyze/stack/stack-details/stack-details.component.html]: This div and everything in it should be indented twice.

File: src/app/analyze/stack/stack-details/stack-details.component.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+  distinct_licenses = '';
+  total_licenses = '';
+
+  total_security_issues = '';
+  cvss = '';
+
+  components_with_tests = '';
+  components_with_dependency_lock_file = '';
+  required_engines = {}
```
Comment (joshuawilson) [inline src/app/analyze/stack/stack-details/stack-details.component.ts]: missing semicolon

File: src/app/analyze/stack/stack-analyses.model.ts (hunk 1 above)
Comment (pmuir) [inline src/app/analyze/stack/stack-analyses.model.ts]: Fixed

File: src/app/analyze/stack/stack-details/stack-details.component.html (hunk 2 above)
Comment (pmuir) [inline src/app/analyze/stack/stack-details/stack-details.component.html]: Fixed

File: src/app/analyze/stack/stack-details/stack-details.component.ts (hunk 3 above)
Comment (pmuir) [inline src/app/analyze/stack/stack-details/stack-details.component.ts]: Fixed

**Pull request merged**